[WARN ] 2019-02-17T17:24:01,673Z [Thread-36 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:60457 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-17T19:12:07,004Z [Thread-14 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:60536 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-17T21:00:10,664Z [Thread-7 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:60604 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-17T21:46:42,012Z [Thread-38 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:60638 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[INFO ] 2019-02-17T22:01:50,973Z [main] hibernate.Version.logVersion - HHH000412: Hibernate Core {1.0}
[INFO ] 2019-02-17T22:01:51,524Z [main] common.Version.<clinit> - HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
[INFO ] 2019-02-17T22:01:51,839Z [main] internal.ConnectionProviderInitiator.initiateService - HHH000130: Instantiating explicit connection provider: net.corda.nodeapi.internal.persistence.HibernateConfiguration$NodeDatabaseConnectionProvider
[INFO ] 2019-02-17T22:01:51,917Z [main] dialect.Dialect.<init> - HHH000400: Using dialect: org.hibernate.dialect.PostgreSQL95Dialect
[INFO ] 2019-02-17T22:01:54,716Z [main] connections.access.getIsolatedConnection - HHH10001501: Connection obtained from JdbcConnectionAccess [org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess@415b83fa] for (non-JTA) DDL execution was not in auto-commit mode; the Connection 'local transaction' will be committed and the Connection will be set into auto-commit mode.
[INFO ] 2019-02-17T22:01:54,863Z [main] internal.JtaPlatformInitiator.initiateService - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
[WARN ] 2019-02-18T00:35:51,061Z [Thread-319 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:62672 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-18T00:35:51,097Z [Thread-319 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:49735 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-18T00:35:51,101Z [Thread-319 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:62674 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-18T00:35:51,116Z [Thread-319 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:62673 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-18T00:35:51,223Z [Thread-421 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@4a593c3c)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:62677 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-18T01:52:35,005Z [Thread-328 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:58629 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-18T02:10:44,322Z [Thread-317 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:59468 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-18T02:53:10,004Z [Thread-372 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:59484 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
[WARN ] 2019-02-18T03:25:58,055Z [Thread-402 (ActiveMQ-server-org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl$5@2fa69580)] core.client.fail - AMQ212037: Connection failure has been detected: AMQ229014: Did not receive data from /127.0.0.1:62526 within the 60,000ms connection TTL. The connection will now be closed. [code=CONNECTION_TIMEDOUT]
